_docs/getting-started/android.md

```diff
@@ -22,11 +22,20 @@ git clone --recursive https://github.com/caffe2/caffe2.git
 
 ## Run the Build Script
 
+If you want to build 32-bit Caffe2:
+
 ```
 cd caffe2
 ./scripts/build_android.sh
 ```
 
+Or if you want to build 64-bit Caffe2:
```

Comment:
Please change to "Or if you want to build Caffe2 for Android with arm64-v8a ABI"